README.md

```diff
@@ -1,113 +1,158 @@
 # Phylo2Vec
 
-This repository contains an implementation of Phylo2Vec. It is distributed under
-the GNU Lesser General Public License v3.0 (LGPL).
+Phylo2Vec is a high-performance software package for encoding, manipulating, and
+analyzing binary phylogenetic trees. At its core, the package contains
+representation of binary trees, which defines a bijection from any tree topology
+with 𝑛 leaves into an integer vector of size 𝑛 − 1. Compared to the traditional
+Newick format, phylo2vec was designed with fast sampling and rapid tree
+comparison in mind.
+
+This current 2.0 version features a core implementation in Rust, providing
+significant performance improvements and memory efficiency while remaining
+available in Python (superseding the version described in the original paper
+([Penn et al., 2024](https://doi.org/10.1093/sysbio/syae030))) and R via
+dedicated wrappers, making it accessible to a broad audience in the
+bioinformatics community.
 
 [![PyPI version](https://badge.fury.io/py/phylo2vec.svg)](https://pypi.org/project/phylo2vec/)
+[![Documentation](https://img.shields.io/badge/docs-latest-blue.svg)](https://phylo2vec.readthedocs.io)
 
-Link to the paper: <https://doi.org/10.1093/sysbio/syae030>
+Link to the paper:
+[https://doi.org/10.1093/sysbio/syae030](https://doi.org/10.1093/sysbio/syae030)
```

Comment:
@Neclow We have the link to the published paper mentioned in https://github.com/uw-ssec/phylo2vec/pull/119/files#r2058878799 here